README.md

```diff
@@ -1 +1,18 @@
-# Twitter-Bot-
\ No newline at end of file
+# Twitter-Bot
+
+## Rate Limits
+The Twitter Bot is limited to pulling 500,000 tweets per month, it is important that we do not reach this limit. Certain endpoints (like filtered stream and recent search) have a limit on how many Tweets they can pull per month. This information can be found via the Twitter Docs. Current usage can be viewed via the Twitter Developer Dashboard for the bot account.
+
+## Running Locally
+Clone this repository and create a file named `.env` in the root of the project. Add and populate the below in this file.
+
+```
+CONSUMER_KEY=
+CONSUMER_SECRET=
+ACCESS_TOKEN=
+ACCESS_TOKEN_SECRET=
+```
+
+This bot can then be run locally by running ```node app.js```
+
+## Deploying
```

Comment:
This is a placeholder until we figure out how we want to handle this

Reply:
I figured out how to handle this, I just have to figure out the best way to do this in an organized manner through Google Cloud, so I will likely complete this as part of a separate task.